cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -235,29 +238,27 @@ def test_course_listing_performance(self):
         self.assertGreaterEqual(iteration_over_courses_time_2.elapsed, iteration_over_groups_time_2.elapsed)
 
         # Now count the db queries
-        with check_mongo_calls(USER_COURSES_COUNT):
+        with check_mongo_calls(150):
             _accessible_courses_list_from_groups(self.request)
 
+        with check_mongo_calls(505):
+            _accessible_courses_list(self.request)
         # Calls:
         #    1) query old mongo
         #    2) get_more on old mongo
         #    3) query split (but no courses so no fetching of data)
-        with check_mongo_calls(3):
```

Comment:
@cpennington Sorry I forgot to mention that its a separate issue. 
[TNL-3866](https://openedx.atlassian.net/browse/TNL-3866)

Also, that this existing test is fetching & counting mongo calls using  `_accessible_courses_list` and `_accessible_courses_list_from_groups` method which do not have our change (`CourseSummary` class)
The method we added which fetch course summaries using `CourseSummary` class is `_staff_accessible_course_list`